### PR TITLE
docs(typescript-plugin): add README

### PR DESCRIPTION
**Pull request by WhyNotHugo**

### CODE REVIEW

File: packages/typescript-plugin/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# typescript plugin
+
+This is a plug-in for `tsserver` or `typescript-language-server`. In must be
+installed in a file-system location visible by the language server or in the
```
Comment (gegoune) [inline packages/typescript-plugin/README.md:4]:
```suggestion
installed in a file-system location accessible by the language server or in the
```

File: packages/typescript-plugin/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# typescript plugin
+
+This is a plug-in for `tsserver` or `typescript-language-server`. In must be
```
Comment (gegoune) [inline packages/typescript-plugin/README.md:3]:
```suggestion
This is a plug-in for `tsserver` or `typescript-language-server`. It must be
```

**Review by gegoune — CHANGES_REQUESTED** [review]

File: packages/typescript-plugin/README.md (hunk 1 above)
Comment (WhyNotHugo) [reply packages/typescript-plugin/README.md]: Fixed

**Comment by johnsoncodehk** [comment]
Thanks!

**Pull request merged**